### ISSUE TITLE
MySQL interactions when buying something in the shop plugin

### ISSUE THREAD

**Issue by MaximusMcfly** [Azuriom/Plugin-Shop#91]
### Description of your suggestion

So when you buy something in the shop like a kit starter it asks you what command should it execute in console (Atleast in Minecraft version of Azuriom it does that), but not all the plugins are designed to be compatible through in-game commands. So i suggest you add a new feature that instead of executing a command in console executes it in MySQL as query command.

### Advantages

Adds support for more plugins and their features.

**Comment by MaximusMcfly** [comment]
Yes, i suggest you make an option while installing? or somewhere in the config files an option to enable it, i think that would make it way more secure

**Comment by crashzk** [comment]
Question, and an API for that, user maintenance API, so that he can create, edit and delete and we only make the call through the game plugin itself, or even through the AzLink plugin.

That way I believe we won't need to force any remote injection into the database. Or would that be impossible?
